Shoko.Server/Models/SVR_VideoLocal_Place.cs

```diff
@@ -398,15 +398,15 @@ public bool RefreshMediaInfo()
             }
         }
         
-        public void RemoveRecordAndDeletePhysicalFile(bool deleteFolder = true)
+        public bool RemoveRecordAndDeletePhysicalFile(bool deleteFolder = true)
```

Comment:
What are all these changes for, if you are wanting to add renamer visibility, this seems to be a lot of irrelevant changes.

Reply:
The PR is for adding everything related to the renamers — including file relocation — and not just renamer viability, hench why these changes and also the file endpoints are in the PR.

Reply:
But we already have had a working file relocation with feedback in APIv1, what is needing refactor?

Reply:
1. The method used in the v1 (see below) was also marked as obsolete and returned both the success status and a potential error message, and the error message was not even used in the parts I edited. So I changed it over to the newer implementation (located right below the older implementation).
  https://github.com/ShokoAnime/ShokoServer/blob/1f943396efd4b9695fdf93bf76ee8877097bf840/Shoko.Server/Models/SVR_VideoLocal_Place.cs#L358-L399
2.The newer implementation (this method) did not return an error message but threw instead, and it also did not return success status when it did not throw. So I added the success status to the newer implementation, but looking closer at it now (and comparing the implementations) I can see that was a mistake. It was fine with a void return.

Also, there is feedback in the new v3 relocate endpoint too, it just throws the error (and lets asp.net serialize it) instead of returning a wrapped object with the error message instead.